python-sdk/tests/files/operators/test_files.py

```diff
@@ -6,7 +6,8 @@
 from astro.files.operators.files import ListFileOperator
 
 
-def test_get_file_list_execute_local():
+@patch("astro.files.locations.base.BaseFileLocation.validate_conn", return_value=None)
```

Comment:
Why do we need to mock `validate_conn`?

Reply:
Can we instead not pass the connection id on L15

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/1478/commits/50bd36d6d59208d166c01b6c00eec416cebc6e15)